sql/ddl/alter/alter-characteristics.md

```diff
@@ -129,7 +131,7 @@ ALTER ??? ???
 ---
 ## Revision
 
-We'd like to set the default value "ruby" to all entries in the column `name` in the `version` table and remove the `NOT NULL` clause to the `id` column.
+We'd like to set the default value `"ruby"` to all entries in the column `name` in the `version` table. We would also like to remove the `NOT NULL` clause to the `id` column.
```

Comment:
This sentence has the right usage of `" "` vs ` ` because `"ruby"` is literally a string value while the others aren't any particular types.